### DISCUSSION TITLE
Exclude Minor Bump From group and use patch instead

### DISCUSSION THREAD

**Discussion by devlsc** [renovatebot/renovate#33167] [Request Help]
### How are you running Renovate?

None

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub v41.0.6

### Please tell us more about your question or problem

Hey,
renovate should create a group PR for minor and patch bumps. We want to exclude the minor bumps for one dependency from the group and only want to include the patch bumps for that dependency in the group. Utilizing the `separateMinorPatch` option results in separate PRs:

* Group PR for minor/patch bumps exclusivly the dependency
* A PR for the minor Bump of the dependency
* A PR for the patch Bump of the dependency (Should be in included in the Group PR)

My current config looks like this:
```{
    "labels": [
        "dependencies", "patch"
    ],
    "prHourlyLimit": 0,
    "prConcurrentLimit": 100,
    "packageRules": [
        {
            "description": "Create Minor and Patch Prs for spring boot",
            "matchPackageNames": [ "org.springframework.boot**" ],
            "separateMinorPatch": true
        },
        {
            "description": "Group minor and patch bumps and auto merge if possible",
            "autoApprove": true,
            "matchUpdateTypes": [
                "patch", "minor"
            ],
            "groupName": "all non-major dependencies versions",
            "groupSlug": "all-minor-patch",
            "automerge": true,
            "automergeType": "pr",
            "platformAutomerge": true
        },
        {
            "description": "PR for spring boot minor",
            "groupName": "Spring boot minor",
            "groupSlug": "spirng-boot-minor",
            "matchPackageNames": [ "org.springframework.boot**" ],
            "matchUpdateTypes": ["minor"]
        }
    ]
}
```

I can't figure out how to include the patch update in the "all non-major dependencies versions" PR.

Every hint and help is appreciated

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment]
I am guessing the current behaviour is that all minor & patch deps are added to the grouped PR `Group minor and patch bumps and auto merge if possible`. And, you want to exclude `minor` updates for the `spring boot` deps from it?

In that case, your config should work fine. Also, the first package rule isn't necessary.

**Reply by devlsc** [reply]
Hey @RahulGautamSingh ,
thanks for the response. 
I tried your recommendation:
```
{
    "labels": [
        "dependencies", "patch"
    ],
    "prHourlyLimit": 0,
    "prConcurrentLimit": 100,
    "packageRules": [
        {
            "description": "Group minor and patch bumps and auto merge if possible",
            "autoApprove": true,
            "matchUpdateTypes": [
                "patch", "minor"
            ],
            "groupName": "all non-major dependencies versions",
            "groupSlug": "all-minor-patch",
            "automerge": true,
            "automergeType": "pr",
            "platformAutomerge": true
        },
        {
            "description": "PR for spring boot minor",
            "autoApprove": true,
            "groupName": "all non-major dependencies versions",
            "groupSlug": "all-minor-patch",
            "automerge": true,
            "automergeType": "pr",
            "platformAutomerge": true,
            "matchPackageNames": [ "org.springframework.boot**" ],
            "matchUpdateTypes": ["patch"]
        },
        {
            "description": "PR for spring boot minor",
            "groupName": "Spring boot minor",
            "groupSlug": "spirng-boot-minor",
            "matchPackageNames": [ "org.springframework.boot**" ],
            "matchUpdateTypes": ["minor"]
        }
    ]
}
```

` And, you want to exclude minor updates for the spring boot deps from it?`
Yes but still want to include the patch bump for the spring dependencies. Unfortunately the solution does not include the patch bump for spring packages in the group PR.
